Not sure why the jenkins vefication test failed with golint error:
```
W1130 10:55:37.200] pkg/storage/etcd3/store_test.go:548:1: context.Context should be the first parameter of a function
W1130 10:55:37.200] 
W1130 10:55:37.200] Please fix the above errors. You can test via "golint" and commit the result.
W1130 10:55:37.200] 
I1130 10:55:37.203] FAILED   hack/make-rules/../../hack/verify-golint.sh	25s
```

Maybe a rebase will solve it.

--- a/pkg/client/cache/reflector.go
+++ b/pkg/client/cache/reflector.go
@@ -259,11 +259,15 @@ func (r *Reflector) ListAndWatch(stopCh <-chan struct{}) error {
 	r.setLastSyncResourceVersion(resourceVersion)
 
 	resyncerrc := make(chan error, 1)
+	cancelCh := make(chan struct{})
+	defer close(cancelCh)
 	go func() {
 		for {
 			select {
 			case <-resyncCh:
 			case <-stopCh:
+				return
+			case <-cancelCh:
 				return
 			}
 			glog.V(4).Infof("%s: forcing resync", r.name)

